### PR TITLE
Append coverage data

### PR DESCRIPTION
**Pull request by felixfontein**
Ref: https://github.com/ansible-community/antsibull-docs-parser/pull/10#issuecomment-1484152686

### CODE REVIEW

File: .github/workflows/nox.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@ jobs:
         with:
           python-versions: "3.7, 3.8, 3.9, 3.10, 3.11"
       - run: |
-          nox -v -e test -p 3.7 3.8 3.9 3.10 3.11 -- --cov-report=xml
+          nox -v -e test -p 3.7 3.8 3.9 3.10 3.11 -- --cov-report=xml --cov-append
       - name: Upload coverage
         uses: codecov/codecov-action@v3
```
Comment (gotmax23) [inline .github/workflows/nox.yml:52]:
```suggestion
        uses: codecov/codecov-action@v3
        with:
          working-directory: antsibull-docs-parser
```

File: .github/workflows/nox.yml [hunk 2]
```diff
@@ -66,6 +66,6 @@ jobs:
         with:
           python-versions: "3.6"
       - run: |
-          nox -v -e test -p 3.6 -- --cov-report=xml
+          nox -v -e test -p 3.6 -- --cov-report=xml --cov-append
       - name: Upload coverage
         uses: codecov/codecov-action@v3
```
Comment (gotmax23) [inline .github/workflows/nox.yml:71]:
```suggestion
        uses: codecov/codecov-action@v3
        with:
          working-directory: antsibull-docs-parser
```

File: .github/workflows/nox.yml [hunk 3]
```diff
@@ -66,6 +68,7 @@ jobs:
         with:
           python-versions: "3.6"
       - run: |
-          nox -v -e test -p 3.6 -- --cov-report=xml
-      - name: Upload coverage
+          nox -v -e test -p 3.6 -- --cov-report=xml --cov-append
         uses: codecov/codecov-action@v3
```
Comment (felixfontein) [inline .github/workflows/nox.yml:72]:
```suggestion
      - name: Upload coverage
        uses: codecov/codecov-action@v3
```

**Comment by gotmax23** [comment]
I did some docs reading and decided to push a different approach. Sorry for the trouble.

**Comment by gotmax23** [comment]
Okay, it seems to be working now. https://app.codecov.io/github/ansible-community/antsibull-docs-parser/commit/b7f914349518f2eeb6848dc8db43fc8ce73a99da/tree/src still shows 100 % coverage and both uploads (`nox-test` and `nox-test-36`) show up.

**Review by gotmax23 — APPROVED** [review]
Ack. I'll let you merge this tomorrow :).

File: .github/workflows/nox.yml [hunk 4]
```diff
@@ -46,10 +46,16 @@ jobs:
         uses: wntrblm/nox@2022.11.21
         with:
           python-versions: "3.7, 3.8, 3.9, 3.10, 3.11"
-      - run: |
-          nox -v -e test -p 3.7 3.8 3.9 3.10 3.11 -- --cov-report=xml
+      - name: Run unit tests and report coverage
```
Comment (felixfontein) [inline .github/workflows/nox.yml:49]:
```suggestion
      - name: Run unit tests
```

**Comment by felixfontein** [comment]
@gotmax23 thanks for your improvements!

**Pull request merged**